recipes/yq/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "yq" %}
+{% set version = "2.4.1" %}
+{% set sha256 = "4833d4055b0f1c1f1a2fd292421b3472da39c7dc2727d7819efd11065a5fd310" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
```

Comment:
Guessing this is because we don't have a Windows build of `jq` yet. Is that correct?

Reply:
Yes, that is correct.

Reply:
xref: https://github.com/conda-forge/jq-feedstock/issues/3